### ISSUE TITLE
Show a Download Progress Bar like the native downloader. 

### ISSUE THREAD

**Issue by aftabaminzoobiapps** [zo0r/react-native-push-notification#1720]
I am developing a project to download videos from social media accounts. all was fine but on android 10 we can not use the native downloader manager. so I want when a user wants to download a video notification will be shown that will have the progress bar, and on the complete download, that notification will be sent that the download has completed.

**Comment by whenmoon** [comment]
Yes, I too would like a progress indicator in the notification like in Notifee for React Native: 

https://notifee.app/react-native/docs/android/progress-indicators

I do own a Notifee license but it does not meet my other requirements.

**Comment by deepbag** [comment]
but the issues is im updating percentage every time notification will  be vibrate every percentage
